### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by MrCoder** [mikeal/watch#149]
Apparently this is one of most popular watches on npmjs.org. I use it a lot myself. I really hope the maintainer can (at least) review PRs and continue this awesome project.

**Comment by flatherskevin** [comment]
@mikeal is this repo still being maintained?

**Comment by Emobe** [comment]
I assume not. The weekly downloads on npm are getting less and less too.

**Comment by arthabus** [comment]
Yeah, @Emobe - I've noticed that too and wondering if there is an alternative to this one which is still maintained?

**Comment by holloway** [comment]
@arthabus try Chokidar https://www.npmjs.com/package/chokidar

**Comment by arthabus** [comment]
thanks @holloway that's exactly what I was looking for! And just for reference, here is the cli version of it as well: https://github.com/kimmobrunfeldt/chokidar-cli

**Comment by nielsonsantana** [comment]
`chokidar-cli` is a perfect replacement.

**Comment by bombillazo** [comment]
> 
> 
> `chokidar-cli` is a perfect replacement.

Thanks! I was looking for something maintained!
